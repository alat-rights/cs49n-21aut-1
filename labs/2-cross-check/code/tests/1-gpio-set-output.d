1-gpio-set-output.o: 1-gpio-set-output.c ../rpi.h
