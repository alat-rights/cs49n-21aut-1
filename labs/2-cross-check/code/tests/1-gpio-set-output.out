TRACE: calling pi code
TRACE: GET32(0x20200008) = 0x66334873
TRACE: PUT32(0x20200008) = 0x66334871
TRACE: pi exited cleanly: 7 calls to random
